frontend/pages/project/[id]/index.tsx

```diff
@@ -22,7 +26,9 @@ export const getServerSideProps = async ({ params: { id }, locale }) => {
 
   // If getting the project fails, it's most likely because the record has not been found. Let's return a 404. Anything else will trigger a 500 by default.
   try {
-    ({ data: project } = await getProject(id, { includes: 'project_images,project_developer' }));
+    ({ data: project } = await getProject(id, {
+      includes: 'project_images,project_developer,country,municipality,department',
```

Comment:
You'll need to add `involved_project_developers` as well, in order to list them in the `<ProjectDevelopers />` section.